docs/PIPELINES-CONFIG.md

```diff
@@ -0,0 +1,271 @@
+
+#### Step 3: Setting up deployment namespaces
+
+You will need to create the namespaces for the different app deployments. With a target namespace called `ai-rhdh`, these namespaces would be called `ai-rhdh-app-development`, `ai-rhdh-app-stage`, and `ai-rhdh-app-prod`. These namespaces can be created using the following:
+
+```sh
+APP_NAMESPACE=$NAMESPACE-app-<developer|stage|prod>
+cat <<EOF | kubectl apply -f - >/dev/null
+apiVersion: v1
+kind: Namespace
+metadata:
+  labels:
+    argocd.argoproj.io/managed-by: $NAMESPACE
+  name: $APP_NAMESPACE
+EOF
+```
+
+#### Step 4: Setting up cosign secret under deployment namespaces
+
+First you will need to fetch the cosign public key from the `signing-secrets` secret in the `openshift-pipelines` namespace that was setup by the installer:
+
+```sh
+kubectl get secrets -n openshift-pipelines signing-secrets -o jsonpath='{.data.cosign\.pub}' 2>/dev/null
+```
+
+Take note of the public key, then use it to create the cosign secret under each of the deployment namespaces:
+
+```sh
+cat <<EOF | kubectl apply -f - >/dev/null
+apiVersion: v1
+data:
+    cosign.pub: <cosign_public_key>
+kind: Secret
+metadata:
+    labels:
+        app.kubernetes.io/instance: default
+        app.kubernetes.io/part-of: tekton-chains
+        operator.tekton.dev/operand-name: tektoncd-chains
+    name: cosign-pub
+    namespace: $APP_NAMESPACE
+type: Opaque
+EOF
+```
+
+#### Step 5: Setting up gitops authentication secret under deployment namespaces
+
+If you are using GitLab, you will need to create the GitLab authentication secret with your GitLab PAT as follows:
+
+```sh
+kubectl -n $APP_NAMESPACE create secret generic "gitlab-auth-secret" \
+    --from-literal=password=<gitlab_pat> \
+    --from-literal=username=oauth2 \
+    --type=kubernetes.io/basic-auth \
+    --dry-run=client -o yaml | kubectl -n $APP_NAMESPACE apply --filename - --overwrite=true >/dev/null
+```
+
+Otherwise, you will need to create a Git authentication secret with your PAT (e.g. GitHub PAT) as follows:
+
+```sh
+kubectl -n $APP_NAMESPACE create secret generic "gitops-auth-secret" \
+    --from-literal=password=<git_pat> \
+    --type=kubernetes.io/basic-auth \
+    --dry-run=client -o yaml | kubectl -n $APP_NAMESPACE apply --filename - --overwrite=true >/dev/null
+```
+
+#### Step 6: Setting up pipelines secret under deployment namespaces
+
+You will need to create a pipeline secret containing the webhook secret for the Git organization for deployments to have proper access via the tekton pipelines:
+
+```sh
+kubectl -n $APP_NAMESPACE create secret generic "pipelines-secret" \
+    --from-literal=webhook.secret=<webhook_secret> \
+    --dry-run=client -o yaml | kubectl -n $APP_NAMESPACE apply --filename - --overwrite=true >/dev/null
+```
+
+#### Step 7: Setting up quay image registry secret under deployment namespaces
+
+For accessing the quay image registry, you'll need to create a secret
```

Comment:
Nit: this sentence shows broken up on 2 different lines

Reply:
Fixed in https://github.com/redhat-ai-dev/ai-rhdh-installer/pull/7/commits/b3f58b158927d574e6df3224dd13a160744a014e